tests/iter/t16076.nim

```diff
@@ -0,0 +1,41 @@
+discard """
+  targets: "c js"
+  output: '''
+0
+10
+'''
+"""
+
```

Comment:
can you wrap in a template main with usual `static: main(); main()` ?

Reply:
fixed